projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -354,6 +354,12 @@ public static function wrap_ui( $callback, $args = array() ) {
 			?>
 			<!-- END OF CALLBACK -->
 
+			<div id="jp-stats-report-bottom">
```

Comment:
Do we need this ID on all old school Jetpack admin pages?

Reply:
No, I did this after I found that I'd need 2 divs in order to make the width of the new section match the rest of the page - and thought that anything that we would want to add to this hook would need this anyway.

But again, It could be all there in the callback. Not a strong preference.